ocaml/xapi/xapi_config.ml

```diff
@@ -37,7 +37,13 @@ let read_config filename =
 		"igd-passthru-vendor-whitelist", Config.Set_string vendor_whitelist;
 		"relax-xsm-sr-check", Config.Set_bool Xapi_globs.relax_xsm_sr_check;
 		"hotfix-fingerprint", Config.Set_string Xapi_globs.trusted_patch_key;
-		"gpumon_stop_timeout", Config.Set_float Xapi_globs.gpumon_stop_timeout;
+		(let cgo = "ciphersuites-good-outbound" in
```

Comment:
Hi @thomassa ,
This change is from below commit:

commit 699ae79d6a1169754abaf4eee1090143dc491f24
Author: Thomas Sanders <thomas.sanders@citrix.com>
Date:   Fri Mar 4 16:06:50 2016 +0000

    CP-14701: Handle ciphersuite config in xapi_config
    
    Fix a build-blocker by adapting a code-change backported from
    the upstream xapi-project/xen-api git repository, which handles
    the ciphersuite config in xapi_globs.ml instead.
    
    Signed-off-by: Thomas Sanders <thomas.sanders@citrix.com>

diff --git a/ocaml/xapi/xapi_config.ml b/ocaml/xapi/xapi_config.ml
index 9a0fd46..e6f9a8d 100644
--- a/ocaml/xapi/xapi_config.ml
+++ b/ocaml/xapi/xapi_config.ml
```
@@ -37,7 +37,13 @@ let read_config filename =
                "igd-passthru-vendor-whitelist", Config.Set_string vendor_whitelist;
                "relax-xsm-sr-check", Config.Set_bool Xapi_globs.relax_xsm_sr_check;
                "hotfix-fingerprint", Config.Set_string Xapi_globs.trusted_patch_key;
-               "gpumon_stop_timeout", Config.Set_float Xapi_globs.gpumon_stop_timeout;
+               (let cgo = "ciphersuites-good-outbound" in
+               cgo, Config.String (fun s ->
+                       D.debug "Processing config %s=%s" cgo s;
+                       Xapi_globs.ciphersuites_good_outbound :=
+                               (if String.filter_chars s (not ++ String.isspace) <> "" then Some s else None)
+               ));
+               "ciphersuites-legacy-outbound", Config.Set_string Xapi_globs.ciphersuites_legacy_outbound;
```